support/modules/single-file

```diff
@@ -10,6 +10,17 @@ dest_dir_file="$FILES"/files/dest_dir
 filename_file="$FILES"/files/filename
 permissions_file="$FILES"/files/permissions
 
+safe_copy() {
+    if [ -n "$3" ]; then
+        echo "safe_copy can only handle one file copy at a time" 1>&2
```

Comment:
Nit:
Is the `1` in `1>&2` reallly necessary?

Reply:
I thought it was, but apparently not. I guess `>&2` is better since `1>&2` looks more similar to `2>&1` (easier to misunderstand). Will change it.